QuizzApp/Core/Constants/QuizzAppColors.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  QuizzAppColors.swift
+//  QuizzApp
+//
+//  Created by bakhva  on 14.06.23.
+//
+
+import UIKit
+
+enum QuizzAppColors {
```

Comment:
ფერებზე გირჩევ, რომ ასეტებში გაიტანო. დარქ მოუდის დასაპორტებისთვისაც მომავალში უკეთესია და შემდეგ გამოყენების ადგილას სახელით მიმართო

Reply:
enum-ის სახელი მხოლობითში უნდა იყოს
```suggestion
enum QuizzAppColor {
```